src/pages/signup-success/index.tsx

```diff
@@ -24,30 +15,42 @@ const Wrapper = styled.section`
     display: flex;
     justify-content: center;
     flex-direction: row;
-    background-color: rgba(200, 214, 215, 0.22);
-
-    @media ${device.mobileL} {
-        padding: 0;
-    }
-`
-
-const StyledDiv = styled.div`
-    padding: 6rem 0;
-    background-color: rgba(200, 214, 215, 0.22);
 
     @media ${device.mobileL} {
         padding: 0;
     }
 `
-const ResponseWrapper = styled.div`
+const ResponseWrapper = styled(Flex)`
     justify-content: center;
-    max-width: 33rem;
+    max-width: 24rem;
     margin: 0 auto;
     flex-direction: column;
     padding: 2rem 1rem;
+    gap: 12px;
+    margin-top: -100px;
+    @media ${device.mobileL} {
+        max-width: 40rem;
+    }
+`
+const EmailLink = styled(StyledLink)`
+    display: table;
+    font-size: 14px;
+    margin-top: 1.8rem;
+    text-decoration: underline;
+    width: 100%;
+    text-align: center;
+`
+
+const query = graphql`
+    query {
+        view_email: file(relativePath: { eq: "sign-up/response-email.png" }) {
+            ...fadeIn
+        }
+    }
```

Comment:
Please don't use this due the in gatsby 5 which coming soon it's not preferable 🙏